bower.json

```diff
@@ -2,11 +2,12 @@
   "name": "paper-ripple",
   "private": true,
   "dependencies": {
-    "core-icon": "Polymer/core-icon#master",
-    "core-icons": "Polymer/core-icons#master",
-    "font-roboto": "Polymer/font-roboto#master"
+    "core-icon": "Polymer/core-icon#0.8-preview",
+    "core-icons": "Polymer/core-icons#0.8-preview",
+    "font-roboto": "Polymer/font-roboto#master",
+    "polymer": "polymer/polymer#0.8-preview"
```

Comment:
Shouldn't demo dependencies be in `dependencies`? The demos are viewable on a production site.